js/tiles.js

```diff
@@ -734,11 +689,7 @@ const forest = {
             this.ground = this.createImage("tiles/rooms/floor/Monster.svg");
         },
         visit: function() {
-            swal({
-                type: 'info',
-                title: 'Hi there',
-                text: "I'm a nameless monster, welcome to my forest. You may find something interesting or equally horrific, let's rely on your fate.",
-            });
+            alertInfo('info', 'Hi there', "I'm a nameless monster, welcome to my forest. You may find something interesting or equally horrific, let's rely on your fate.");
```

Comment:
`this.alertInfo('Hi there', "I'm a nameless monster, welcome to my forest. You may find something interesting or equally horrific, let's rely on your fate.")`

Reply:
So, how will people mention the type of the Swal?